src/Orleans/Core/GrainClient.cs

```diff
@@ -277,11 +277,22 @@ public static void Uninitialize()
         }
 
         /// <summary>
+        /// Test hook to uninitialize client without cleanup
+        /// </summary>
+        public static void HardKill()
```

Comment:
Shouldn't this be an internal method? Having this in public surface of a type that is normally used is not the best. I would be OK if this is public but in a different type that users are not typically exposed to, but of course I prefer internal if possible,